test/unit/sdam/srv_polling.test.ts

```diff
@@ -110,7 +109,7 @@ describe('Mongos SRV Polling', function () {
         expect(loggerError).to.have.been.calledOnceWith(
           sinon.match(/Unexpected MongoRuntimeError/)
         );
-      });
+      }).skipReason = 'TODO(NODE-4817): determine whether or not to keep this here';
```

Comment:
Do we need these tests?

Reply:
I think this can be removed now and once we add the new logger we can add new tests. It's pretty poorly named anyways.